HISTORY.rst

```diff
@@ -2,6 +2,24 @@
 History
 =======
 
+3.1.0 (2025-09-21)
```

Comment:
Since there are breaking changes, this is gonna be a 4.0.0.

    4.0.0 (unreleased)

I might even issue a beta release, if only to setup PyPI publishing action.

I'll deal with this later on.